CONTRIBUTING.rst

```diff
@@ -356,23 +369,22 @@ silence any warnings generated when running the tests.
 Deprecation policy
 ------------------
 
-A feature release may deprecate certain features from previous releases. If a
-feature is deprecated in feature release 1.A, it will continue to work in all
-1.A.x versions (for all versions of x) but raise warnings. Deprecated features
-will be removed in the first 1.B release, or 1.B.1 for features deprecated in
-the last 1.A.x feature release to ensure deprecations are done over at least 2
-feature releases.
+A MAJOR and MINOR releases may deprecate certain features from previous
+releases. If a feature is deprecated in MINOR release A.x, it will continue to
```

Comment:
This sentence mentions only about MINOR release. But there are no mentions about MAJOR release. How about using "in a release" instead?